types/node/worker_threads.d.ts

```diff
@@ -73,7 +73,12 @@ declare module "worker_threads" {
         postMessage(value: any, transferList?: Array<ArrayBuffer | MessagePort>): void;
         ref(): void;
         unref(): void;
-        terminate(callback?: (err: Error, exitCode: number) => void): void;
+        /**
+         * Stop all JavaScript execution in the worker thread as soon as possible.
+         * Returns a Promise for the exit code that is fulfilled when the `exit` event is emitted.
+         * @param callback Deprecated, use promise instead.
+         */
+        terminate(callback?: (err: Error, exitCode: number) => void): Promise<number>;
```

Comment:
`terminate()` no longer accepts a callback now

Reply:
It's deprecated, not entirely removed but useless regardless.
We can remove it to encourage people to migrate however.